censusreporter/apps/census/templates/profile/profile_detail.html

```diff
@@ -97,7 +97,9 @@ <h1 class="title">{{ geography.this.short_name }}</h1>
     </div>
 </section>
 {% endcomment %}
-
+<h2>This applicaiton is currently under active development.<br>
+Please provide any feedback you may have through the dialogue at the bottom right of your screen.</h2>
```

Comment:
Ditto for dialog -> button details, more understandable, because there ain't no dialog until you hit the button.